src/fides/api/models/connectionconfig.py

```diff
@@ -148,6 +148,13 @@ class ConnectionConfig(Base):
         uselist=False,
     )
 
+    pre_approval_webhooks = relationship(  # type: ignore[misc]
+        "PreApprovalWebhook",
+        back_populates="connection_config",
+        cascade="delete",
+        uselist=False,
```

Comment:
I think we should remove `uselist=False` because a connection config can have multiple pre approval webhooks right? If uselist=False, this will just return one object.

Reply:
Model-level tests are good for flushing these out -

Reply:
Ahhh I see, that makes sense! I'll remove this

Reply:
This means I likely should add `uselist=False` to the `privacy_request` field on my `PreApprovalWebhookReply` api model: 
```
privacy_request = relationship(
        "PrivacyRequest", back_populates="pre_approval_webhook_reply", uselist=False  # type: ignore
    )
```